lib/Default/SmrForce.class.inc

```diff
@@ -89,27 +93,35 @@ class SmrForce {
 		}
 	}
 
-	protected function __construct($gameID,$sectorID, $ownerID) {
+	protected function __construct(&$gameIDOrResultArray, $sectorID, $ownerID) {
 		$this->db = new SmrMySqlDatabase();
-		$this->db->query('DELETE FROM sector_has_forces WHERE expire_time < ' . $this->db->escapeNumber(TIME));
```

Comment:
Where is expiration of forces handled now?

Reply:
`SmrForce::getForce` calls `tidyUpForces`
https://github.com/smrealms/smr/blob/9ec5382c97d4a3aa5e8667f297cfa402f9167fee/lib/Default/SmrForce.class.inc#L70
and `tidyUpForces` does the same deletion query that was performed in the ctor
https://github.com/smrealms/smr/blob/9ec5382c97d4a3aa5e8667f297cfa402f9167fee/lib/Default/SmrForce.class.inc#L88

(Note that all calls to the `SmrForce` ctor go through `getForces`.)